### ISSUE TITLE
Check IPv6 Only Sites

### ISSUE THREAD

**Issue by shohruh** [RealFaviconGenerator/realfavicongenerator#75]
I have a site http://ipv6.uzstyle.com/. This site is only reachable via IPv6. http://realfavicongenerator.net/favicon_checker?site=ipv6.uzstyle.com checking my site, but have errors..

**Comment by shohruh** [comment]
Please, check.

**Comment by phbernard** [comment]
Oh... I noticed the issue a week ago on my phone, then I forgot it... Sorry.

I tried to access http://ipv6.uzstyle.com/ with my browser but it didn't work... I never used IPv6, but as far as I know, I rely on my ISP to get IPv6 support, is that right? What surprises me is that I'm even not able to resolve the domain name ipv6.uzstyle.com (whereas uzstyle.com works). It seems to me that this has nothing to do with IPv6. Any clue?

**Comment by shohruh** [comment]
This site is only reachable via IPv6. Yes, you need to get support IPv6 on your ISP. But, you can get IPv6 support via Teredo (http://en.wikipedia.org/wiki/Teredo_tunneling).

**Comment by jacksonj04** [comment]
IPv6 is a whole bundle of hurt at the moment - it's the future, but most ISPs and other providers (like hosting providers) are taking their sweet time getting there. `ipv6.uzstyle.com` does exist and is resolving as expected, try `dig ipv6.uzstyle.com AAAA` to check the AAAA record.

For this to work the hosting provider which is currently looking after RealFaviconGenerator will need to support IPv6 connectivity, and the server it's on will need to be configured to resolve IPv6 addresses as well. _In theory_ once this is done the actual application won't need changing at all, save for perhaps curl or whatever else you're using to actually retrieve content being told that it's safe to try IPv6 resolution.

**Comment by phbernard** [comment]
This is as complex as I expected :) Being more developer than sysadmin, this is not a task which I feel comfortable with. Sorry @shohruh, I'm afraid this issue will remain opened for some time :-/
